src/WebUI/Common/KubeSummary.tsx

```diff
@@ -252,7 +252,7 @@ export class KubeSummary extends BaseComponent<IKubeSummaryProps, IKubernetesCon
         const selectedItemType = this.state.selectedItemType;
         // ToDo :: Currently for imageDetails type, the selected item will be undefined, hence adding below check. Remove this once we have data from imageService
         if (selectedItemType
-            && !(selectedItem && ([SelectedItemKeys.ServiceItemKey, SelectedItemKeys.OrphanPodKey, SelectedItemKeys.PodDetailsKey].some(s => s == selectedItemType)))
+            && (selectedItem || !([SelectedItemKeys.ServiceItemKey, SelectedItemKeys.OrphanPodKey, SelectedItemKeys.PodDetailsKey].some(s => s == selectedItemType)))
```

Comment:
make this as const value
!([SelectedItemKeys.ServiceItemKey, SelectedItemKeys.OrphanPodKey, SelectedItemKeys.PodDetailsKey].some(s => s == selectedItemType)

outside of this if and then use that const, also do youw ant to use ===